test/phoenix/tracker/shard_replication_test.exs

```diff
@@ -5,14 +5,26 @@ defmodule Phoenix.Tracker.ShardReplicationTest do
   @primary :"primary@127.0.0.1"
   @node1 :"node1@127.0.0.1"
   @node2 :"node2@127.0.0.1"
+  @node3 :"node3@127.0.0.1"
 
   setup config do
     tracker = config.test
-    {:ok, shard_pid} = start_shard(name: tracker)
-    {:ok, topic: to_string(config.test),
-     shard: shard_name(tracker),
-     shard_pid: shard_pid,
-     tracker: tracker}
+
+    conf = [
+      topic: to_string(tracker),
+      shard: shard_name(tracker),
+      tracker: tracker
+    ]
+
+    conf =
+      if config[:primary] != false do
```

Comment:
This is actually needlessly complex, 
we can configure extra tracker opts via a tag,
so instead of `primary: false`, we'd have `tracker_opts: [permdown_period: 2000]` as a tag 
and in the test replace node1 with primary and node3 with node1.